.github/workflows/check_format_code.yml

```diff
@@ -7,16 +7,7 @@ on:
   pull_request:
     branches:
       - "main" 
+
 jobs:
-  formatting-check:
-    name: Formatting Check
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2
-    - name: Run bootstrap
-      run: cd tools/ && ./bootstrap
-    - name: Run clang-format style check for C/C++ programs.
-      uses: jidicula/clang-format-action@v4.0.0
-      with:
-        clang-format-version: '12'
-        check-path: '.'
+  check_code_format:
```

Comment:
Minor nit but let's fix it please: `check_code_format` but the file is called `check_format_code`, can we make everything be `check_code_format` please? Thanks!

Reply:
Sure! Great catch. I will make it be `check_code_format` in `dev-tools` too.